ducktape/command_line/main.py

```diff
@@ -200,12 +224,26 @@ def main():
         JUnitReporter(test_results),
         FailedTestSymbolReporter(test_results)
     ]
+=======
+    runner = TestRunner(cluster, session_context, session_logger, tests)
+
+    # Report results
+    reporters: list[SummaryReporter] = []
+    if test_results:
+        reporters += [
+            SimpleStdoutSummaryReporter(test_results),
+            SimpleFileSummaryReporter(test_results),
+            HTMLSummaryReporter(test_results),
+            JSONReporter(test_results),
+            JUnitReporter(test_results)
+        ]
+>>>>>>> 80360b1 (main: Add debugging for hung tests)
```

Comment:
nit: merge?

Reply:
probably should be removed no?

Reply:
Yep.. missed merge conflict marker, thanks.